nakamoto-upgrade/signing-and-stacking/stacking-flow.md

```diff
@@ -725,30 +725,34 @@ The CLI command is:
 
 ```bash
 stacks-signer generate-stacking-signature \
-  --pox-address bc1... \
+  --method stack-stx \
+  --max-amount 1000000000000 \
+  --auth-id 195591226970828652622091037492597751808 \
```

Comment:
where does this auth-id come from?

Reply:
The source code states that it is:

> A unique identifier to prevent re-using this authorization

The docs:

> a random integer that prevents re-use of a particular signature, similar to how nonces are used with transactions

I generated it through the command also listed there:

```bash
openssl rand -hex 16 | awk '{ print strtonum("0x"$0) }'
```

I am torn on whether the code snippet should provide examples that the executable would accept, instead of placeholders like in the `--help` for the CLI that force the user to change them. Especially for the fields we are sure the user will have to customise and/or that might provide security troubles.

I can change that to a placeholder, like `--auth-id <AUTH_ID>`. What do you think?

Reply:
Also, I am considering suggesting generating the `nonce` (if we have to) with this:

```bash
python3 -c 'import secrets; print(secrets.randbits(128))'
```

It relies on Python's standard library only and does not go into edge-cases of whether `gawk` correctly handles big numbers (spoiler, it probably doesn't and we would need at least to add `-M` to its invocation).

Reply:
both of those are valid options, i just think it's important to specify *what* this field needs, either as a comment or explanation before the command example. 
I see your addition below, thanks!